examples/advanced/stochastic_odes.py

```diff
@@ -59,7 +59,7 @@ def exact_solution(t, dt):
 cols = ceil(1.0 * len(dts) / rows)
 errors = dict([(method, zeros(len(dts))) for method in methods])
 for dt_idx, dt in enumerate(dts):
-    print 'dt: ', dt
+    print('dt:', dt)
```

Comment:
My preference would probably be (3), but (2) is nice too when it's a simple example like that (maybe (2) when simple and (3) when more complicated?).